.github/workflows/ci-xqts.yml

```diff
@@ -22,8 +22,9 @@ jobs:
         env:
           JAVA_OPTS: -XX:+UseG1GC -XX:+UseStringDeduplication
         run: find exist-xqts/target -name exist-xqts-runner.sh -exec {} --xqts-version HEAD --output-dir /tmp/xqts-output \;
+      - name: Check XQTS Logs
+        run: ls /tmp/xqts-output 
```

Comment:
so this means...... (1) no directory is created or (2) no files in the directory?

Reply:
`ls` will fail with exit code 2 if the directory is not found. If the directory is created but has no contents it will output `.` and `..` I believe. So there is room for improvement.

Reply:
Maybe `ls /tmp/xqts-ouput/**/*.xml` is what we really want. As we expect the zip-file to contain
a subfolder with a bunch of test results in XML files.

Reply:
But since I am unsure what the actual contents of that folder will be I am hesitant to go this step right away.